src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -132,6 +132,12 @@ def get_asic_id_from_name(asic_name):
     Returns:
         asic id.
     """
+    #Exception addition
+    if asic_name is None:
```

Comment:
The 'check' and the exception error message is bit confusing. Instead of 'AttributeError', raise 'ValueError'